src/doom/d_pwad.c

```diff
@@ -183,8 +170,151 @@ void D_LoadSigilWad (void)
 		}
 	}
 
-	// [crispy] regenerate the hashtable
-	W_GenerateHashTable();
+	return true;
+}
+
+// [crispy] auto-load Sigil II
+static boolean LoadSigil2Wad (const char *iwaddir, boolean pwadtexture)
+{
+    int i, j;
+    const char *sigil2_basename;
+    char *autoload_dir;
+
+    const char *const sigil2_wads[] = {
+        "SIGIL_II_MP3_V1_0.WAD",
+        "SIGIL_II_V1_0.WAD",
+    };
+
+    static const struct {
+        const char *name;
+        const char new_name[8];
+    } sigil2_lumps [] = {
```

Comment:
The intermission music seems to depend on how the Sigil wads are loaded:
1. If autoloaded, `SIGIL_II_V1_0.WAD` is loaded first, then `SIGIL_v1_21.wad`, so the first Sigil's intermission music is used in Sigil II.
2. If sideloaded, the correct intermission music is used for each Sigil episode.
3. If `-file` is used, the intermission music depends on the load order, one replacing the other.

Similar behavior occurs with the other lumps, like the end pic `SIGILEND` and the demos. Not sure if any of this needs to be fixed, just something I noticed.

Reply:
Probably best to leave it be.